app/models/template.rb

```diff
@@ -469,25 +482,6 @@ def new_family_id
     family_id
   end
 
-  # Default values to set before running any validation
-  def set_defaults
-    self.published ||= false
-    self.archived ||= false
-    self.is_default ||= false
-    self.version ||= 0
-    unless id?
-      self.visibility = if (org.present? && org.funder_only?) || is_default?
-                          Template.visibilities[:publicly_visible]
-                        else
-                          Template.visibilities[:organisationally_visible]
-                        end
```

Comment:
I've removed this from the defaults for now as we set this in the `templates_controller#create` method, and it would not necessarily have access to the `org` attribute at creation time